python/ray/serve/_private/constants.py

```diff
@@ -49,6 +49,12 @@
     os.environ.get("MAX_DEPLOYMENT_CONSTRUCTOR_RETRY_COUNT", "20")
 )
 
+# Max retry on deployment constructor is
+# min(num_replicas * MAX_PER_REPLICA_RETRY_MULTIPLIER, MAX_DEPLOYMENT_CONSTRUCTOR_RETRY_COUNT)
+MAX_PER_REPLICA_RETRY_MULTIPLIER = int(
```

Comment:
Since the name contains `per_replica` already, `count` seems like a simpler name, wdyt
```suggestion
MAX_PER_REPLICA_RETRY_COUNT = int(
```